patches/spatialite.patch

```diff
@@ -6,10 +6,34 @@ diff -rupN spatialite/src/gaiaaux/gg_utf8.c spatialite-ios/src/gaiaaux/gg_utf8.c
  #if defined(__APPLE__) || defined(__ANDROID__)
```

Comment:
Is this patch applied automatically?

Reply:
Yep: https://github.com/iosphere/libspatialite-ios/blob/master-moers/update-spatialite#L3

Reply:
Nice :)